.github/workflows/release_wasm_fdw.yml

```diff
@@ -68,7 +68,13 @@ jobs:
           CHECKSUM: ${{ steps.extract.outputs.CHECKSUM }}
         run: |
           cat > README.txt <<EOF
-          To use this Wasm foreign data wrapper on Supabase, create a foreign server like below,
+          To use this Wasm foreign data wrapper on Supabase, create a foreign table like below,
+
+          create extension if not exists wrappers with schema extensions;
```

Comment:
will this be formatted as code?

Reply:
nope, it's just a plain text file included in the release package.